plugins/modules/podman_login.py

```diff
@@ -116,6 +134,21 @@ def login(module, executable, registry, authfile,
         changed = True
         if 'Existing credentials are valid' in out:
             changed = False
+        # If we have managed to calculate a checksum before, check if it has changed
+        # due to the login
+        if checksum:
+            if os.path.exists(authfile):
```

Comment:
same here, if we found a required file before, let's just use it:

```python
if checksum and os.path.exists(check_file):
    content = open(authfile, 'rb').read()
  ....
```

Reply:
We don't need to check for the existence of the file since without its existence, we couldn't have calculated the checksum. I've implemented the change.